nextflow_schema.json

```diff
@@ -87,7 +87,7 @@
                 "tools": {
                     "type": "string",
                     "fa_icon": "fas fa-toolbox",
-                    "description": "Tools to use for variant calling and/or for annotation.",
+                    "description": "Tools to use for duplicatemarking, variant calling and/or for annotation.",
```

Comment:
I am not super happy about this, because of how we enable spark, seems slightly inconsistent

Reply:
how about we do it the way we use to, and have a general --sentieon boolean flag, that will automatically takes for mapping and duplicate marking the sentieon tools (I don't see people using sentieon bwa and not wanting to use sentieon dedup), and adds the sentieon_dnaseq tools?